ui/src/app.tsx

```diff
@@ -20,30 +20,17 @@ const prepackagedConceptSets = [
   {
     id: "_demographics",
     name: "Demographics",
-    entity: "person",
+    occurrence: "",
   },
   {
     id: "_analgesics",
     name: "Analgesics",
-    entity: "ingredient_occurrence",
+    occurrence: "ingredient_occurrence",
     filter: {
-      relationshipFilter: {
-        outerVariable: "ingredient_occurrence",
-        newVariable: "concept",
-        newEntity: "ingredient",
-        filter: {
-          binaryFilter: {
-            attributeVariable: {
-              variable: "concept",
-              name: "concept_id",
-            },
-            operator: tanagra.BinaryFilterOperator.DescendantOfInclusive,
-            attributeValue: {
-              int64Val: 21604253,
-            },
-          },
-        },
-      },
+      type: FilterType.Classification,
+      occurrenceID: "ingredient_occurrence",
+      classificationID: "ingredient",
+      keys: [21604253],
```

Comment:
Why do you need this specific ingredient concept id here?

Reply:
This is a prepackaged concept set and that's the `concept_id` for Analgesics. This was a temporary location for these and I can relocate them to the underlay at this point. I'll do that in another PR though.